docs/examples/input/mixed-input.vue

```diff
@@ -30,12 +30,12 @@
 </template>
 
 <script lang="ts">
-import { defineComponent, ref } from 'vue'
+import { defineComponent, ref, shallowRef } from 'vue'
 import { Search } from '@element-plus/icons'
 export default defineComponent({
   setup() {
     return {
-      Search,
+      Search: shallowRef(Search),
```

Comment:
There should be no need for use shallowRef

Reply:
Yeah, setup can return plan object, besides, we still have `raw()` method, we don't need this extra step here.